backend/api/views/main.py

```diff
@@ -24,6 +24,30 @@ def get_persons():
     return create_response(data={"persons": persons})
 
 
+# function that is called when you visit /census_response
+@main.route("/census_response", methods=["GET"])
+def get_census_response():
```

Comment:
could you make the endpoint name more descriptive of what the query & params are?